docs/introducing/history.rst

```diff
@@ -7,127 +7,122 @@ History
-for Wireguard, unicast VRRP, RPKI for BGP and fully 802.1ad-compliant
-QinQ ethertype. The telnet server and support for P2P filtering were
-removed.
+Crux introduced many new features, some of the most noteworthy are: 
+an mDNS repeater, a broadcast relay, a high-performance PPPoE server, 
+an HFSC scheduler, and support for Wireguard, unicast VRRP, RPKI for BGP, 
+and fully 802.1ad-compliant QinQ ethertype. The telnet server and support 
+for P2P filtering were removed.
 
-Crux is the first version to feature the modular image build system.
-CLI definitions began to be written in the modern, verifiable XML
-templates. Python APIs were introduced for command scripting and
-configuration migration. Introduction of new Perl and shell code was
-proscribed and the rewriting of legacy Perl code in pure Python began
-with Crux.
+Crux was the first VyOS release to feature a modular image build system.
+CLI definitions were written using modern, verifiable XML templates. 
+Python APIs were introduced for command scripting and
+configuration migration. The introduction of new Perl and shell code was
+prohibited, and the process of rewriting legacy Perl code in pure Python 
+began with Crux.
 
-Crux reached end of support in 2023..
+Crux reached the end of support in 2023.
 
 Equuleus (1.3)
 --------------
 
-The current long-term support version of VyOS, Equuleus (the Pony)
-came out on 21 December 2021, once again in time for the winter
-holidays.
+Equuleus (the Pony) was a long-term support version released 
+on 21 December 2021, just in time for the winter holidays.
 
-Equuleus brought many long-desired features with it, most notably
-an SSTP VPN server, an IPoE server, an OpenConnect VPN server and
-a serial console server, in addition to reworked support for WWAN
-interfaces, support for GENEVE and MACSec interfaces, VRF, IS-IS
-routing, preliminary support for MPLS and LDP, and many other
-initialisms.
+Equuleus brought many long-awaited features, most notably an SSTP VPN 
+server, an IPoE server, an OpenConnect VPN server, and a serial console 
+server. It also introduced reworked support for WWAN interfaces, support 
+for GENEVE and MACSec interfaces, VRF, IS-IS routing, and preliminary support 
+for MPLS and LDP.
 
-Equuleus reached end of support in 2025.
+Equuleus reached the end of support in 2025.
 
 Sagitta (1.4)
 -------------
 
-Sagitta (the Arrow) was released in 2024 and is currently a supported LTS release.
+Sagitta (the Arrow), released in 2024, is currently a supported LTS release.
 
 Circinus (1.5)
 --------------
 
-Circinus (the Compass) is the codename of the upcoming development
-branch, so there's no VyOS 1.5 yet.
+Circinus (the Compass) is the codename for the upcoming development
+branch. VyOS 1.5 Circinus has not been released yet.
 
 A note on copyright
 ===================
 
-Unlike Vyatta, VyOS never had (nor will ever have) proprietary code.
+Unlike Vyatta, VyOS has never had proprietary code and never will.
```

Comment:
```suggestion
Unlike Vyatta, VyOS has never had closed-source code and never will.
```

The reality is that we may have to use something like AGPL or BSL for some components in the future to keep the balance between making the source available for inspection and modification and being able to keep the project afloat. So I'd shift the focus to the closed/available source distinction here.